testsuite/tests/lf_skiplist/stubs.c

```diff
@@ -131,10 +131,9 @@ inline static uintnat calc_value(uintnat id) { return id; }
 inline static uintnat calc_key(uintnat id,uintnat turn) { return 1024*id+turn+1; }
 inline static uintnat calc_right(uintnat id,uintnat turn,uintnat ndoms) { return (id+turn) % ndoms; }
 
-CAMLprim value insert_skiplist(value turn_val,value ndoms_val,value domain_id_val) {
-  CAMLparam3(turn_val,ndoms_val,domain_id_val);
+CAMLprim value insert_skiplist(value turn_val,value domain_id_val) {
+  CAMLparam2(turn_val,domain_id_val);
   uintnat domain_id = Long_val(domain_id_val);
-  uintnat ndoms = Long_val(ndoms_val);
```

Comment:
Would it suffice to remove the declaration and computation of `ndoms`, leaving `ndoms_val` as an unused function parameter?  This would result in a smaller patch overall, and perhaps better preserve the intent of the original code ("any of these stubs functions may need to known the number of domains").  But it assumes that neither `-Wunused-parameter` nor `-Wunused` are in force.

Reply:
It is given `-Wall -Werror=format-security`.  This is with gcc 13.1.1.

Reply:
OK, thanks for the info.  In this case the simpler fix I outlined above should suffice.  I took the liberty to push it on this branch (see 1c513eb457).